include/abb_librws/common/rw/retcode.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <string>
+
+namespace abb::rws::rw::retcode
+{
+    /**
+     * \brief A structure for containing response of recode resource.
+     */
+    struct RetcodeInfo
+    {
+        int code;                /** \brief The error code or success code as number */
```

Comment:
I agree the `code` in `RetcodeInfo` struct is redundant when using `getRetcodeInfo()` and `getRetcodes()` functions but I think it's nice to have this struct representing the XML fields of response. 
It will be helpful to reuse this code if we'd like to pass this retcode struct to some other function.
It's also necessary if we would like to have the `to_string` and `ostream::operator<<` print whole diagnostic message with the retcode not just the description, so I wouldn't remove it.